src/renderer/store/project.js

```diff
@@ -106,6 +106,9 @@ const mutations = {
 const actions = {
   LISTEN_FOR_LOAD_PROJECT ({ commit, dispatch }) {
     ipcRenderer.on('AGANI::open-project', (e, { pathname, name }) => {
+      // Initialize editor and show empty/new tab
+      dispatch('NEW_BLANK_FILE')
```

Comment:
Why need to show an empty tab?

Reply:
Because if you open a folder the editor, titlebar (functionality to move the window, controls, ...) and theme is not initialized. In other word the titlebar and editor are not yet created.

![mt_not_init](https://user-images.githubusercontent.com/22716132/53903753-c9da6e00-4044-11e9-85da-f7c2e1a9fa4b.gif)


The first window is the default window if you start the application. The second window when opening a folder (`File -> Open Folder`) and, as you can see, titlebar and editor are missing. If you click inside Mark Text the editor etc are created. So my solution is to explicitly create/initialize the  editor etc and show the empty tab.

Reply:
how about open the first(open the last opened files in the future feature) markdown file in the directory? open a new blank tab when there is no markdown file in the opened directory.

Reply:
I think an empty tab is sufficient. Most editor do so and we can later replace an empty/unchanged tab with a new one.

Reply:
OK!